src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -19,6 +22,8 @@ import { Column } from '../../../../utils/analysis-utils';
 function AnalysisTable({ classes, tableData, columns }: AnalysisTableProps) {
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [sortColumn, setSortColumn] = useState('');
```

Comment:
Consider typing this with `Column['id']` (see later comment)

Reply:
done